pymetis/__init__.py

```diff
@@ -373,7 +373,7 @@ def part_graph(nparts, adjacency=None, xadj=None, adjncy=None,
                       eweights, options, recursive)
 
 
-def part_mesh(n_parts, connectivity, options=None, tpwgts=None, gtype=None):
+def part_mesh(n_parts, connectivity, options=None, tpwgts=None, gtype=None, ncommon=1):
```

Comment:
Please document the parameter.

Reply:
OK, after I add the doc, should I create a new pull request?

Reply:
No, just push to the existing branch.

Reply:
The doc and test example are added now